services/prefect/flows/generated_flows.py

```diff
@@ -97,46 +94,6 @@ def create_flow_for_scraper(ix: int, cls: Type[DatasetBase], schedule=True):
     return flow
 
 
-def create_cdc_single_state_flow():
```

Comment:
Removing the CDCCovidDataTracker removes the need for this flow. This would create a task to run `CDCCovidDataTracker` for each state.

Reply:
Ah, yeah...  It would be nice to delete this.  But I don't think we should, at least until we hear back from CMU.  😦

Reply:
Very good call, I had forgotten about that group. I'll re-add this!